cpu/stm32/periph/ptp.c

```diff
@@ -61,10 +61,10 @@
 #endif /* !STM32_PTPSSIR */
 
 /**
- * @brief   Return the result of x / y, scientifically rounded
+ * @brief   Return the rounded result of x / y (rounded as in not cropped or truncated)
```

Comment:
```suggestion
 * @brief   Return the rounded result of x / y
```

Let's keep the `@brief` brief :-) Feel free to squash

Reply:
k